src/http/validations/device.js

```diff
@@ -13,6 +13,19 @@ const getSingleDevice = (req, res, next) => {
   return next();
 };
 
-export default {
-  getSingleDevice,
+const updateDevice = (req, res, next) => {
+  const { deviceId } = req.params;
+  const { label } = req.body;
+
+  const fieldErrors = new FieldErrors();
+
+  if (!deviceId || typeof (deviceId) !== 'string') fieldErrors.addError('deviceId', 'deviceId is a required string');
+  if (!label || typeof (label) !== 'string') fieldErrors.addError('label', 'label is a required string');
```

Comment:
I don't think it should be a must to provide the `label` field when updating a device